### PR TITLE
10516 - cleanup the dtui in-page nav component

### PR DESCRIPTION
**Pull request by anjenkin**
### Author
- [x] Story Book has been rebuilt (new build artifacts in `/docs`)
- [x] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [ ] All new components have been exported in `/index.js` `(if applicable)`
- [ ] Design Review Complete
- [ ] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [ ] Integration Status Updated on relevant documentation page `(if applicable)`
- [ ] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [ ] Code review

### CODE REVIEW

File: helpers/inPageNavHelper.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * inPageNavHelper.js
+ * Created by Andrea Blackwell 01/16/24
+ */
+import {useCallback} from "react";
```
Comment (brianpetway) [inline helpers/inPageNavHelper.js:5]: looks like this isn't used

File: helpers/inPageNavHelper.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+export const checkIsOverflow = (ulEl, padding) => {
+    let left = false;
+    let right = false;
+    console.log(ulEl, padding);
```
Comment (brianpetway) [inline helpers/inPageNavHelper.js:10]: a few console logs in this block

File: jest.config.js [hunk 3]
```diff
@@ -4,14 +4,14 @@ module.exports = {
...
     collectCoverage: true,
-    coverageThreshold: {
-        global: {
-            branches: 80,
-            functions: 80,
-            lines: 80,
-            statements: -10
-        }
-    },
+    // coverageThreshold: {
```
Comment (brianpetway) [inline jest.config.js:7]: did you mean to leave this commented out?

**Comment by brianpetway** [comment]
A few inline comments.

The testing strategy is very clever, that must have taken a minute to arrive at.
I know you said you didn't get to add as many tests as you had hoped; in the future we should add tests for different screen sizes, with the different numbers for padding, since we've had issues around that

**Comment by anjenkin** [comment]
Yeah I could see the tests I added being helpful during development given the issues with calculating the overflow but not sure how helpful this is now.  we could add test for the padding issue in another ticket.

**Review by brianpetway — APPROVED** [review]

**Pull request merged**